examples/LoRaWAN/LoRaWAN_TS_Packages/LoRaWAN_TS009.h

```diff
@@ -0,0 +1,236 @@
+#include <Arduino.h>
+#include <RadioLib.h>
+// #include <RadioBoards.h>
+
+#warning "The variables below must match your main code. Please check your radio type!"
+extern SX1278 radio;          // this can be any LoRaWAN-compatible type (e.g. SX1262)
+extern LoRaWANNode node;
```

Comment:
Instead of making all these variables `extern`, it should be possible to just put the declarations here, as the .ino file already includes this header.